Ds 321 input button submit (#304)
bifrost-vui/bifrost-front-end-library: CHANGELOG.mdx

### Input Radio
- new [Input Radio](/story/components-input-radio--drupal) twig component

### Button

> - Improved [Twig Button Component](/story/components-button--drupal) to handle submit functionalities.

Reviewer: ```suggestion
- Improved [Twig Component Button](/story/components-button--drupal) to handle submit functionalities.
```

Author: j'ai des doute sur la qualité de la syntaxe.

Reviewer: Comment ça? Quels doutes?

Reviewer 2: hors sujet pour la PR

### Input Radio
- new [Input Radio](/story/components-input-radio--drupal) twig component

### Button

> - Improved [Twig Button Component](/story/components-button--drupal) to handle submit functionalities.
> - New arguments are available: `isSubmit`, `isAutofocus`, `id`, `formId`, `formAction`, `formEncryptionType`, `formTarget`, `formMethod` , `formNoValidate`

Reviewer: ```suggestion
- New properties to handle submit functionalities on [Twig Component Button](/story/components-button--drupal). `isSubmit`, `isAutofocus`, `id`, `formId`, `formAction`, `formEncryptionType`, `formTarget`, `formMethod` , `formNoValidate`.
```
